### ISSUE TITLE
aria_roledescription and headings

### ISSUE THREAD

**Issue by brennanyoung** [accessibilitysupported/a11ysupport.io#224]
I notice that my BigSur is ignoring `aria-roledescription` for headings, whether implemented as ARIA or HTML, although it **does** work for the button used in the test case here. 

This suggests that support for `aria-roledescription`on Safari/VO is overstated in the data, which only addresses the button case. They may be other semantics that fail too.

Should we expand the example to include a few more roles, maybe a lot more, or make another example for headings?

Failing code (semantic announced as "heading level 2" by VoiceOver):

`<h2 aria-roledescription="headline">This is where it all happens</h2>`
